src/testrel.jl

```diff
@@ -633,17 +638,25 @@ function _test_rel_step(
         #   If an abort is expected it is encountered
         #   If no abort is expected it is not encountered
         #   If results are expected, they are found (other results are ignored)
-        #   If problems are expected, they are found (other problems are ignored)
-        #   If no problems are expected, warning level problems are ignored
+        #   If problems are expected, they are found
+        #   Unexpected problems with severity worse than allowable are not found
 
         unexpected_errors_found = false
+        error_levels = []
```

Comment:
since we're testing membership with this - maybe a set is preferable? (though at this size of data, probably not making a notable perf difference either way)

Reply:
I tested, and error level comparison with a set of Symbols was faster by ~9% with similar memory use. In less than a thousand runs of the tests we would gain a second of testing time using a set. The set also better represents the unique error levels it contains so I changed it to a set.